Makefile

```diff
@@ -78,29 +79,41 @@ dist-test: $(MANIFEST)
 	ANSIBLE_COLLECTIONS_PATHS=build/collections ansible-doc $(NAMESPACE).$(NAME).organization | grep -q "Manage Organization"
 
 $(MANIFEST): $(NAMESPACE)-$(NAME)-$(VERSION).tar.gz
+ifeq ($(COLLECTION_COMMAND),mazer)
+	# No idea, why this fails. But mazer is old and deprecated so unlikely to beeing fixed...
+	# mazer install --collections-path build/collections $<
+	-mkdir build/collections build/collections/ansible_collections build/collections/ansible_collections/theforeman build/collections/ansible_collections/theforeman/foreman
+	tar xf $< -C build/collections/ansible_collections/theforeman/foreman
+else
```

Comment:
Somehow, this seems to be enough to make ansible 2.8 happily execute the tasks for `filters` and `katello_manifest` all the modules requiring the `foreman_helper` fail.
Please anyone with a decent knowledge about "ansible 2.8 & mazer" speak up!

Reply:
I'm not sure I like being put in the "decent mazer knowledge" camp, but I might have fixed it.

Reply:
That is why i hesitated to ping you.